userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -95,6 +104,28 @@ Fixed Term:: A type of <<Phase,phase>> that specifies an end date and during whi
 
 == I
 
+Invoice:: An invoice is a bill provided to a customer for charges that are payable on the customer's account. A single invoice consolidates charges for all subscriptions held by that customer for a specified time period. Invoices may be set to be paid automatically, or may be paid manually by the customer. An invoice consists of one or more invoice items.
+
+Invoice Item:: Represents a single item charged on an invoice. Given an active subscription, one could see multiple items for that subscription on a single invoice including recurring items, usage items, fixed price items, etc. There can also be items for different subscriptions on the same invoice, as well as items that are unrelated to subscriptions such as adjustments and taxes. 
+
+
+Invoice Item Type:: Specifies the type of invoice item. Can be one of `EXTERNAL_CHARGE`, `FIXED`, `RECURRING`, `REPAIR_ADJ`, `CBA_ADJ`, `ITEM_ADJ`, `USAGE`, `TAX`, `PARENT_SUMMARY`. Refer to the
+https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[_Subscription Billing_] document to know more. 
+
+Invoice Payment:: Payment made against a specific invoice. By default, Kill Bill attempts to make a payment against an invoice using the account's default payment method. 
+
+Invoice Refund:: Refund amount associated with an invoice. Kill Bill allows issuing a full or partial refund against an invoice payment. 
+
+Invoice Amount:: The amount that a customer is charged as part of an invoice. Also known as "Invoice Total",  "Total",  "Charged Amount" or simply "Amount". At a high level, it is calculated by adding the amounts on all the chargeable invoice items associated with an invoice. See https://docs.killbill.io/latest/invoice_examples.html#_charged_amount[charged amount] to know more.
+
+Invoice Balance:: The amount that the customer owes as part of an invoice. It is positive if the customer owes something or zero otherwise. At a high level,  it is calculated by subtracting the paid amount from the charged amount. See https://docs.killbill.io/latest/invoice_examples.html#_invoice_balance[invoice balance] to know more.
+
+Invoice Credit:: The amount that we owe to a customer as part of an invoice.
```

Comment:
```suggestion
Invoice Credit:: The amount that your business owes to a customer as part of an invoice.
```

Reply:
This could benefit from an example that explains why an amount might be credited on an invoice (like incorrectly billed or customer contested amount).

Reply:
Made suggested changes and added an example, let me know how it looks

Reply:
Looks good!